parlai/messenger/core/agents.py

```diff
@@ -74,6 +75,12 @@ def put_data(self, message):
             }
             self.msg_queue.put(action)
 
+    def set_state_data(self):
```

Comment:
Considering this is to some degree just _linking_ the agent's state data and the manager's state data on this agent, I'd suggest just calling it in the `__init__` function and initializing this data field to an empty dict on the AgentState initialization. This way it's more clearly an alias to the shared data pool between all instances of ParlAI agents from this particular user. As now it's somewhat unclear (I'd also suggest a more generic name, as anything can be put here, perhaps `stored_data` or `persistent_data`?)

Reply:
Good call! I'll make the changes and call it `stored_data'.